src/drive/drive-gateway.ts

```
@@ -0,0 +1,380 @@
+
+    const index = url.getParam("index");
+    if ('undefined'!== typeof index) {
+      name += `-${index}`;
+    }
+    name += ".fp";
+    
+    try{
+      const fileId = await search(this.logger, '', name, auth);
+      if( 'undefined' !== typeof fileId && fileId !== 404){
+        response = await get(this.logger, fileId, 'fileContent', auth);
+        var contentArray = new Array(response.length);
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)